Compile errors with `--all-features` on macOS

```sh
cargo clippy --workspace --all-targets --all-features
```

```
error[E0433]: failed to resolve: could not find `selinux` in `uucore`
   --> src/uu/id/src/id.rs:145:36
    |
145 |         selinux_supported: uucore::selinux::is_selinux_enabled(),
    |                                    ^^^^^^^ could not find `selinux` in `uucore`
```

```
error[E0433]: failed to resolve: could not find `smack` in `uucore`
   --> src/uu/id/src/id.rs:198:27
    |
198 |             match uucore::smack::get_smack_label_for_self() {
    |                           ^^^^^ could not find `smack` in `uucore`
```

```
error[E0425]: cannot find value `uumain` in crate `chcon`
   --> target/debug/build/coreutils-dbc4be9239a5e04a/out/uutils_map.rs:152:27
    |
152 |         ("chcon", (chcon::uumain, chcon::uu_app)),
    |                           ^^^^^^ not found in `chcon`
```

etc

```text
$ cargo clippy --workspace --all-targets --all-features -q --keep-going
error[E0425]: cannot find value `uumain` in crate `chcon`
   --> target/debug/build/coreutils-f72911ea23e264cf/out/uutils_map.rs:152:27
    |
152 |         ("chcon", (chcon::uumain, chcon::uu_app)),
    |                           ^^^^^^ not found in `chcon`

error[E0425]: cannot find value `uu_app` in crate `chcon`
   --> target/debug/build/coreutils-f72911ea23e264cf/out/uutils_map.rs:152:42
    |
152 |         ("chcon", (chcon::uumain, chcon::uu_app)),
    |                                          ^^^^^^ not found in `chcon`

error[E0425]: cannot find value `uumain` in crate `runcon`
   --> target/debug/build/coreutils-f72911ea23e264cf/out/uutils_map.rs:214:29
    |
214 |         ("runcon", (runcon::uumain, runcon::uu_app)),
    |                             ^^^^^^ not found in `runcon`

error[E0425]: cannot find value `uu_app` in crate `runcon`
   --> target/debug/build/coreutils-f72911ea23e264cf/out/uutils_map.rs:214:45
    |
214 |         ("runcon", (runcon::uumain, runcon::uu_app)),
    |                                             ^^^^^^ not found in `runcon`
```

--- a/.github/workflows/code-quality.yml
+++ b/.github/workflows/code-quality.yml
@@ -72,7 +72,7 @@ jobs:
       matrix:
         job:
           - { os: ubuntu-latest  , features: all             , workspace: true }
-          - { os: macos-latest   , features: feat_os_unix }
+          - { os: macos-latest   , features: all             , workspace: true }
           - { os: windows-latest , features: feat_os_windows }
           - { os: ubuntu-latest  , features: feat_wasm , target: wasm32-wasip1 }
     steps:

